manifests/init.pp

```diff
@@ -148,6 +154,9 @@
   Variant[Stdlib::Absolutepath, Array[Stdlib::Absolutepath]] $pulpcore_additional_import_paths = [],
   Variant[Stdlib::Absolutepath, Array[Stdlib::Absolutepath]] $pulpcore_additional_export_paths = [],
   Boolean $pulpcore_telemetry = false,
+
+  Optional[Stdlib::Fqdn] $hostname = undef,
```

Comment:
Making it undef by default means that if the FQDN changes, the user gets new certificates. This has implications on the hostname rename script.